VocaDbWeb.Core/Controllers/Api/ResourcesApiController.cs

```diff
@@ -0,0 +1,105 @@
+#nullable disable
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Resources;
+using AspNetCore.CacheOutput;
+using Microsoft.AspNetCore.Mvc;
+using VocaDb.Web.Resources.Domain.ReleaseEvents;
+using ApiController = Microsoft.AspNetCore.Mvc.ControllerBase;
+
+namespace VocaDb.Web.Controllers.Api
+{
+	/// <summary>
+	/// Loads localized string resources.
+	/// </summary>
+	[Route("api/resources")]
+	[ApiController]
+	// TODO: implement [DefaultCasingConfig]
+	public class ResourcesApiController : ApiController
+	{
+		// TODO: implement
+		/*class DefaultCasingConfig : Attribute, IControllerConfiguration
+		{
+			public void Initialize(HttpControllerSettings controllerSettings, HttpControllerDescriptor controllerDescriptor)
+			{
+				controllerSettings.Formatters.Clear();
+				controllerSettings.Formatters.Add(new JsonMediaTypeFormatter());
+				controllerSettings.Formatters.JsonFormatter.SerializerSettings.ContractResolver = new DefaultContractResolver();
+			}
+		}*/

```

Comment:
Why do we need this?

Reply:
If it's already using camelCase, then no need. I think that's the default now. Probably was some legacy code.